.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,67 @@
+# For most projects, this workflow file will not need changing; you simply need
```

Comment:
Is this file addition related to the translation? If not, can we remove it from this PR? Thanks!

Reply:
@ramaazharywibisana 

>